frontend/isoko/src/components/BusinessPreview.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Rating } from 'react-simple-star-rating';
+import { Table } from 'react-bootstrap';
+import MinorityTag from './MinorityTag';
+import KeywordTag from './KeywordTag';
+
+const BusinessContainer = styled(Table)`
+   width: 85%;
+   height: 25%;
+   background-color: #fff;
+   border: 1px solid #cecece;
+   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+   min-height: 20vh;
+   min-width: 60vh;
+   margin-bottom: 3%;
+`;
+
+const Photo = styled.img`
+   height: 100%;
+   width: 100%;
+   object-fit: cover;
+`;
+
+const RightContainer = styled(Table)`
+   justify-content: flex-start;
+   display: flex;
+   flex-grow: 1;
+   flex-direction: column;
+   padding-left: 0.25em;
+`;
+
+const Text = styled.div`
+   font-size: 0.75rem;
+   display: flex;
+   flex-shrink: 1;
+   padding-bottom: 0.5em;
+`;
+
+const Title = styled.h2`
+   align-self: left;
+   font-size: 1rem;
+   margin: auto 5px auto 0px;
+   margin-right: 18px;
+   margin-top: 15px;
+   margin: none;
+   margin-block-start: 0em;
```

Comment:
I think some of these margins are cancelling eachother out. 

I would suggest using margin shorthand for this type of scenario 
https://www.w3schools.com/css/css_margin.asp